crypto/ts/ts_rsp_verify.c

```diff
@@ -494,29 +494,33 @@ static int ts_check_status_info(TS_RESP *response)
 }
 
 static char *ts_get_status_text(STACK_OF(ASN1_UTF8STRING) *text)
+{
+    return sk_ASN1_UTF8STRING2text(text, '/');
+}
+
+char *sk_ASN1_UTF8STRING2text(STACK_OF(ASN1_UTF8STRING) *text, char sep)
```

Comment:
Now you've made this generic, it no longer seems appropriate that this function sits in crypto/ts

Reply:
I've moved this function definiition to `crypto/asn1/asn1_lib.c`.